### DISCUSSION TITLE
Get-PnPUnifiedAuditLog + Power Automate Events

### DISCUSSION THREAD

**Discussion by robcaretta-thrivent** [pnp/powershell#3114] [Q&A]
Is it possible to pull power automate related events via this command?  If so, could someone provide an example?
https://pnp.github.io/powershell/cmdlets/Get-PnPUnifiedAuditLog.html

**Comment by AndersRask** [comment]
According to this article Power Automate is part of the unified log

https://learn.microsoft.com/en-us/microsoft-365/compliance/audit-log-search?view=o365-worldwide

There are actually also a sample in PowerShell that uses Exchange Online PowerShell, so that would also be a viable approach if you do not get entries using the pnp cmdlet
https://learn.microsoft.com/en-us/microsoft-365/compliance/audit-log-search-script?view=o365-worldwide

**Reply by robcaretta-thrivent** [reply]
The cmdlet Search-UnifiedAuditLog is unstable and unreliable. 
I was hoping someone would have an example using Get-PnPUnifiedAuditLog.
